### ISSUE TITLE
Net Scan: 4TB All, 40TB BAB&app_vars

### ISSUE THREAD

**Issue by jspenguin2017** [uBlockOrigin/uAssets#4268]
20 packages scanned, 1 package processed, about 50,000+ packages expected per month. 
About 80 GB data scanned out of about 200+ TB expected per month. 

<details>

Hard anti-adblock: 
```
# BAB
http://f4.motogon.ru/motokross/mxgp-2018/msg717172/
https://dc-chronicle.com/2018/08/01/robert-mueller-refers-democrat-tony-podesta-criminal-charges/
https://www.sofiotheque.info/2018/02/telecharger-50-dossiers-de-maladies.html
# Other
http://www.kizi.cm/    # Comes up every time
```

Annoyance: 
```
# anOptions
http://allamericansthings.com/2018/11/prototyping-the-betentacled-inflatable-soft-robots-of-zero-gee/
http://thejewishvoice.com/2016/03/16/intel-acquires-replay-technologies-to-up-its-sports-game/
https://usa.watchpro.com/millennials-reason-behind-luxury-spending-boost-china/
http://www.ebizlatam.com/tag/gerardo-coronel/
http://www.fagagnaonline.com/
https://2spendless.com/?tag=forever
https://centrafriqueactu.com/2018/11/01/congo-la-gouvernance-forestiere-au-coeur-dun-forum-regional-a-brazzaville/
https://www.betglob.pl/tag/33
```

</details>

**Comment by jspenguin2017** [comment]
This scanner is just a initial prototype, I hacked it together within 30 minutes, I'll write proper automation scripts later.

**Comment by okiehsch** [comment]
I can*t consistently reproduce the BAB entries, never had that issue.

**Comment by okiehsch** [comment]
`ebizlatam.com` is fixed in the regional list.

**Comment by okiehsch** [comment]
@jspenguin2017 let's use this issue as reference for your future scan results.

**Comment by jspenguin2017** [comment]
OK, I'll close this for now, I'll reopen when new data become available.

**Comment by jspenguin2017** [comment]
```
Possible BlockAdBlock: https://audienciapixelada.com/videos/270
Possible BlockAdBlock: https://bestmobs.co/g-tide-e90/
Possible BlockAdBlock: http://bihar.result91.com/akupatna/mba-2nd-sem-exam-result-2018/88158
Possible BlockAdBlock: http://www.breakingworldnewstoday.com/2013/05/1568.html
Possible BlockAdBlock: https://business.dailynews.us.com/2018/05/we-still-don-understand-social-security.html
```

**Comment by jspenguin2017** [comment]
6~10
```
Possible BlockAdBlock: https://www.ccvgaming.com/forum/awards.php?s=a92de730df0b9b0dfae5bf3a3a8ea49a
Possible BlockAdBlock: https://www.centergeek.it/miui-si-aggiorna-alla-versione-2-3-16/
Possible BlockAdBlock: http://choone.com/?view=ads&catid=7&subcatid=113&cityid=372&lang=fa
Possible BlockAdBlock: http://freeclassifiedads.qtellads.com/index.php?view=post&postevent=&cityid=160&lang=en&catid=&subcatid=
Possible BlockAdBlock: https://infodifesa.it/corte-dei-conti-illegittima-ritenuta-sulla-pensione-a-generale-della-guardia-di-finanza/
```

**Comment by jspenguin2017** [comment]
11~15
```
Possible BlockAdBlock: https://www.isolaillyon.it/2014/06/17/le-bizzarre-avventure-di-jojo-stardust-crusaders-parte-iii.html
Possible BlockAdBlock: https://library.avsim.net/esearch.php?DLID=160208&Name=&FileName=&Author=&CatID=
Possible BlockAdBlock: https://www.magazine-advertisements.com/braun-tassimo-coffee-maker.html
Possible BlockAdBlock: https://www.marmitedumonde.com/tag/set-de-couteaux-a-pizza
Possible BlockAdBlock: http://multicoinfaucet.cf/doge/?cc=reCaptcha
```

**Comment by jspenguin2017** [comment]
16~20
```
Possible BlockAdBlock: http://netherlands.marcyads.com/0/posts/194-Sports-and-Fitness-/4234--Fitness-for-Sale/3436-buy-phentermine.html
Possible BlockAdBlock: https://passivenation.com/Thread-07-10-2018-socks-5-4--4394
Possible BlockAdBlock: https://pornslay.com/blowjobs-fotos/
Possible BlockAdBlock: http://www.queen68.gr/2016/12/blog-post_84.html
Possible BlockAdBlock: http://www.savemydinar.com/2016/11/cbk-kuwait-enjoy-10-discount-on-crumbs.html
```

**Comment by okiehsch** [comment]
Well, atleast one site includes multiple anti-adblock scripts, I will fix such sites seperately and reference this issue.

**Comment by jspenguin2017** [comment]
```
https://www.dearjulius.com/2018/11/3-nutrition-lessons-to-steal-from.html
https://mobil.igpure.com/tools
https://www.marketmovers.it/2009/10/provvedimento-finarte-casa-daste-spa.html
http://www.mrhowtosay.com/view/pol/rus/2008942
```

Popup? 
```
http://paytoshi.in/?cc=reCaptcha
```

**Comment by jspenguin2017** [comment]
I scanned 30 more packages, but only found 4 BAB. Looks like my initial estimate was way off. 

Also, I don't know whether the domains will repeat in later packages. I need to update my scanner to keep track of it. I also want my scanner to be able to handle AdFly, IL, and more. 

On top of this, I need to find a way to drain the [backlog](https://github.com/NanoMeow/QuickReports/tree/master/net-scan-backlog), so I'm shutting down my server for now.

**Comment by okiehsch** [comment]
`http://paytoshi.in/?cc=reCaptcha` I can reproduce a popup, no anti-adblock message on my end.

**Comment by jspenguin2017** [comment]
Yes, only popup. Closing for now, I need to think about the next step.

**Comment by smed79** [comment]
> no anti-adblock message on my end.

Go to `http://paytoshi.in/?cc=reCaptcha`
Click "[**You are Human**](https://i.imgur.com/vHg0VeM.png)".
The page will redirect you to `http://payco.xyz/FButmCYnOojY` ==> anti adblock

**Comment by okiehsch** [comment]
I worked through the first 100 lines of `net-scan-backlog/anOptions0.txt` and `net-scan-backlog/anOptions1.txt`.

````
2spendless.com
allamericansthings.com
bookwonderland.com
dailynexus.com
centrafriqueactu.com
ebizlatam.com
homasg.com
kizi.cm
lentes-et-poux.fr
moonbunnycafe.com
salient.org.nz
socialbookmarkings.us
softmaker.kz
thejewishvoice.com
usa.watchpro.com
vodoleyworld.ru
````
are either already fixed, can't reproduce the anti-adblock messsage, can't connect or parked domains.
I will add the rest.

**Comment by jspenguin2017** [comment]
18 out of 200, that's not an acceptable ratio, I'll definitely need to find a way to automate it, at least partially.

**Comment by jspenguin2017** [comment]
Annoyance: 
```
https://www.airsoft-milsim-news.com/sightmark-mini-shot-m-spec-fms-red-dot-sight/
https://anglais-pdf.com/exercice-grammaire-anglais-muchmany/
http://blog.j172.tw/%E7%B4%B3%E5%A3%AB%E7%9A%84%E5%BF%83%CB%99%E7%97%9E%E5%AD%90%E7%9A%84%E6%83%85/%E7%95%B6%E9%A1%98%E6%9C%9B%E5%AF%A6%E7%8F%BE%E4%BA%86/
https://centrafriqueactu.com/2018/10/31/gouvernance-forestiere-le-premier-forum-du-cv4c-souvre-a-brazzaville/
https://chelorg.com/2018/11/05/tusk-has-arrived-to-warsaw-for-questioning-about-the-financial-pyramid/
```

**Comment by jspenguin2017** [comment]
Annoyance
```
http://www.cogitoergo.it/junk-silver-quantita-totali-coniate-negli-usa-dalla-fine-del-%E2%80%99700-ad-oggi/
https://www.der-bank-blog.de/ratgeber/sparbrief-zinsen-erklaert-wer-kann-davon-profitieren/2442/?utm_source=bankblog
https://www.designtagebuch.de/t-online-portal-erneuert/
http://dialogos.ba/2018/05/25/ibn-taymiyyina-promisljanja-o-zekatu-placanju-u-naturi-ili-novcu-i-sl/
```

Hard Anti-adblock
```
https://currentaffairs.gktoday.in/union-govt-approves-a-project-national-optical-fibre-network-nofn-project-0420135910.html
```

**Comment by jspenguin2017** [comment]
Annoyance
```
https://earmilk.com/2017/08/02/river-returns-with-kitsune-on-midnight-x/
http://www.ebizlatam.com/tag/paula-garces/
http://ecijabpeinfo.com/?tag=puente-diciembre
```

The rest (unverified, about 80% accurate): https://github.com/NanoMeow/QuickReports/blob/master/net-scan-backlog/anOptions-filtered.txt

**Comment by okiehsch** [comment]
I already added `currentaffairs.gktoday.in` , like I said yesterday

> currentaffairs.gktoday.in uses admiral

**Comment by jspenguin2017** [comment]
Ops, looks like I forgot to update my filters...

**Comment by okiehsch** [comment]
https://github.com/uBlockOrigin/uAssets/issues/4268#issuecomment-444748012
`ebizlatam` is fixed in the regional list.

**Comment by jspenguin2017** [comment]
OK, added to global scanner whitelist.

**Comment by mapx-** [comment]
removed duplicate https://github.com/uBlockOrigin/uAssets/commit/bb1b3e4d2194addb7ebe89ce50c65229f0917d61

**Comment by okiehsch** [comment]
I removed `https://centrafriqueactu.com/2018/10/31/gouvernance-forestiere-le-premier-forum-du-cv4c-souvre-a-brazzaville` from https://github.com/uBlockOrigin/uAssets/issues/4268#issuecomment-445389074, already fixed in annoyances.

**Comment by jspenguin2017** [comment]
Eh... My automation scripts still need improvements... 
I'll have it load annoyance filter as well.

**Comment by okiehsch** [comment]
`der-bank-blog.de` has to go in `uBO-filters` the message reappears after every pageload.

**Comment by okiehsch** [comment]
Same with `dialogos.ba`.

**Comment by jspenguin2017** [comment]
Hard (NSFW)
```
http://adultscribe.com/live-cams/camsoda/the%20horny%20hostel%20-%20bedroom
```

Hard
```
https://www.amni8.com/2018/06/samsung-galaxy-j3-2018.html
https://www.baby-magnum.review/kokoro-connect-review-summer-2012-alur-cerita-yang-keren/
https://bestsellerforaday.com/black-wedding-dresses-meaning/
# Comes up every time
https://br.nacaodamusica.com/posts/dashboard-confessional-we-fight/
```

**Comment by jspenguin2017** [comment]
Hard + Popup + NSFW
```
http://gaypornwave.com/dmitry-osten-andro-maas/
```

Hard
```
https://homedesignnearme.com/25148/small-apartment-interior-design-mumbai/
https://jalasenastri.com/tag/ngipri-atau-ngepet/
# Comes up every time, may need 1~2 refreshes
http://www.hagalil.com/2015/03/muenchen-tipps-2/
# Comes up every time
https://komionline.ru/news/21328
```

**Comment by jspenguin2017** [comment]
Hard
```
http://lemontreetours.com/chaise-haute-bar-pas-cher/
https://luxurydreamhomes.net/tag/acrylic-vs-enameled-steel-bathtubs/
http://majorpress.net/be-well-f-i-t-flow-restorative-yoga/4/
http://meraswasthyameriaawaz.org/how-to-make-a-wonderful-birthday-card/how-to-make-a-wonderful-birthday-card-new-lovely-birthday-card-collage-tellmeladwp/
http://miltonfriedmancores.org/college-resume-essay-examples/college-resume-essay-examples-15-fresh-resume-portfolio-website-template-for-resume-best-college-resume/
```

**Comment by jspenguin2017** [comment]
Hard
```
https://minutolivre.com/globo-escolhe-miguel-falabella-para-comentar-o-oscar-2017/
https://www.novahomeimprovementsmd.com/tag/antique-white-kitchen-cabinets-for-sale/
# Comes up every time
https://www.onlineclassnotes.com/
```

Annoyance
```
http://mywrestling.com.pl/author/wojtekczentorycki/
https://ocen-piwo.pl/Casablanca_Premium_Beer%2C1%2C928
```

**Comment by jspenguin2017** [comment]
Hard
```
http://www.piazzagallura.org/santa-teresa-gallura-ot-officineperegrine-teatro-presenta-il-progetto-i-racconti-del-cuscino/
https://rumahfit.com/taman-sempit-dengan-sayuran-di-dalam-pot/
http://tattoostyles.us/tag/tattoo-ideas-unusual
http://www.vivianeaudet.com/cr-z-%E4%B8%AD%E5%8F%A4/
# Comes up every time
http://www.orologidapolso.info/contatti/
```

**Comment by jspenguin2017** [comment]
--- anOptions backlog ---

Hard 
```
https://www.elquintobeatle.com/tag/su-nombre-real-es-otro/
https://www.en.magicgameworld.com/quake-champions-how-to-find-lore-scrolls/
```

Annoyance
```
https://www.essentialhomme.fr/star-wars-the-last-jedi-nouvelle-bande-annonce-devoilee/
https://www.euribor.com.es/2018/11/05/una-sala-de-28-magistrados-del-supremo-delibera-en-estos-momentos-quien-debe-pagar-el-impuesto-sobre-hipotecas/
https://gptimes.de/fallout-76-geschichten-aus-den-huegeln-von-west-virginia-die-begegnung-in-flatwoods/
```

**Comment by jspenguin2017** [comment]
Annoyance
```
https://gripped.com/
https://healthyfitcare.com/tag/receding-gums-cure/
https://konserwatyzm.pl/niemieccy-lewacy-wybieraja-sie-do-warszawy-na-marsz-niepodleglosci/
https://www.leblogducinema.com/tag/alice-ferney/
```

Hard
```
https://www.ilprocidano.it/prestigiacomo-vivara/immagine-092/
```

**Comment by jspenguin2017** [comment]
Annoyance
```
http://www.midcenturyhome.com/3-eichler-renovations-that-will-leave-you-speechless/
https://www.mrfreeat33.com/resources/
https://www.myandroidsolutions.com/category/solution/
https://pbn.com/titleist-doesnt-find-lewd-golf-parody-funny/
```

Hard
```
# Comes up every time
https://www.pacbiztimes.com/section/economy/east-ventura-county/page/22/
```

**Comment by okiehsch** [comment]
`usa.watchpro.com` is already part of annoances.

**Comment by jspenguin2017** [comment]
OK, looks like I somehow missed it. 
OK, pushed an entry to quick reports tracker for further observation.

**Comment by jspenguin2017** [comment]
I now have fingerprinting rule for AdFly, Admiral, anOptions, app_vars, BAB, FAB, and IL. What else should I add?

**Comment by jspenguin2017** [comment]
OK, thanks for the info, I updated my scanner. 

Also I forgot to attach IAM role to my server again... I have the attention span of a goldfish...

**Comment by jspenguin2017** [comment]
Also I'm scanning pre-processed archives, so I don't need to worry about redirections.

**Comment by okiehsch** [comment]
I mentioned it for the benefit of anybody who reads this thread, I had some complaints about people following links and then getting malware alert messages.
Maybe we should put a diclaimer "follow link at your own risk" in  the `readme`. :wink:

**Comment by jspenguin2017** [comment]
RIP...

<details>

![image](https://user-images.githubusercontent.com/7283682/49681561-2687b600-fa61-11e8-99f1-499d9652e6dc.png)

</details>

**Comment by jspenguin2017** [comment]
Annoyance

```
http://www.noodletowntranslated.com/global-evolution/global-evolution-chapter-45/?replytocom=9108
```

**Comment by okiehsch** [comment]
I thought your server was down.

**Comment by jspenguin2017** [comment]
Hard (BAB)
```
http://www.baumblaetter.de/
https://furkankhan.000webhostapp.com/bookmarking/design-build-in-manhattan-apartment-renovations-nyc-design-build-firm-in-nyc.html
http://gameinfo.pw/pendapatan-grand-theft-auto-v-kalahkan-penjualan-film-avatar/
```

**Comment by jspenguin2017** [comment]
Well, not down enough I guess...

**Comment by jspenguin2017** [comment]
Hard
```
http://www.classifiedscalgary.ca/ads/calgary-toronto-real-estate-ad-700854/
http://www.metropolisweb.it/metropolisweb/2018/05/17/scavi-pompei-torna-alla-luce-vicolo-dei-balconi/
https://www.mocasoft.ro/
```

Annoyance
```
http://mywrestling.com.pl/mycast-6-special-qa-with-shadow-juz-20-lipca-kanale-mywrestling/
```

**Comment by jspenguin2017** [comment]
Hard
```
https://www.upi.com/Top_News/US/2018/06/21/House-passes-farm-bill-tightening-work-requirements-for-SNAP/9081529619198/?spt=rrs&or=5
```

Annoyance
```
https://webwereld.nl/e-commerce/4478-einde-news-nl-is-niet-het-einde-van-de-scanpen
```

**Comment by okiehsch** [comment]
`https://webwereld.nl/e-commerce/4478-einde-news-nl-is-niet-het-einde-van-de-scanpen`
What kind of generic anti-adblock solution got flagged on that site?

**Comment by jspenguin2017** [comment]
Admiral.

**Comment by okiehsch** [comment]
Yes it is, I can see a request to an admiral domain, but that is blocked by Peter Lowe's list and even if I allow it, there is no anti-adblock message.

**Comment by jspenguin2017** [comment]
I see a red banner: 
![image](https://user-images.githubusercontent.com/7283682/49689737-b3268880-fae2-11e8-95d1-cac262ff041d.png)

**Comment by okiehsch** [comment]
My reply "Yes it is" meant, yes it is caused by something other than admiral.
Fixed now.

**Comment by jspenguin2017** [comment]
Annoyance
```
https://www.centurylife.org/
https://comoinvertirenbitcoin.co/
https://www.paolobruno.net/acquista-doraemon-dvd-al-miglior-prezzo-su-ebay/
```

Hard
```
https://crazytron.net/?ref=valera83
# One refresh required to trigger
https://www.scuolaon.it/liceo-giacomo-leopardi-recanati/
```

**Comment by jspenguin2017** [comment]
Hard
```
https://www.tedamo.de/telekommunikation/early-adopter-torsten-seiler-verstaerkt-online-konzeption/
https://www.hackintosh.download/discover/
# Comes up every time
https://philippinenmagazin.de/tag/computer/
# Need refresh & takes a while
https://www.pcwelt.de/ratgeber/So_belauschen_Hacker_Sie_ueber_Ihre_Webcam-IT-Sicherheit-7915977.html
```

**Comment by jspenguin2017** [comment]
That's a solid half a terabyte of data analyzed. 

The last 50 packages only yielded 14 true positives. Considering that my server needs about 16 minutes to process one package once it run out of CPU credits, I need to look into potential optimizations...

**Comment by okiehsch** [comment]
````
https://www.adproceed.com/ad/the-zblackcard-is-the-most-powerful-prep-paid-debit-card-in-north-america/
````
Well, it is neither FAB or admiral, looks like anOptions to me.

**Comment by jspenguin2017** [comment]
Well, I guess they tried all of them and was captured in the corpus.

**Comment by okiehsch** [comment]
What get's flagged at `pcwelt.de`? I can't reproduce any anti-adblock message, I followed your steps.

**Comment by jspenguin2017** [comment]
It was Admiral. 
Let me check.

**Comment by jspenguin2017** [comment]
![image](https://user-images.githubusercontent.com/7283682/49691242-09072a80-fafb-11e8-8002-40e9701e89f6.png)

**Comment by okiehsch** [comment]
Ok, I was testing in incognito mode, if I use a normal session I can reproduce, but it is certainly not admiral and does not look like any generic solution at all.

**Comment by jspenguin2017** [comment]
`||tinypass.com^` works, but I'm not sure if it breaks anything. 
Yea it's not Admiral.

**Comment by mapx-** [comment]
`@@||pcwelt.de/js/advert.js$script,first-party`

**Comment by jspenguin2017** [comment]
I think I just need a bigger server.

**Comment by jspenguin2017** [comment]
Alright, I finished scanning about 4 TB of data, it costed $2.45, so no, the current pure brute-force approach isn't going to work. I need to spend some time to properly design a cost-efficient scanning engine that can take advantage of multiple CPU cores and spot instances. 

<details>

![image](https://user-images.githubusercontent.com/7283682/49898834-462f2d80-fe17-11e8-9263-eaf6edcba22d.png)

</details>

**Comment by jspenguin2017** [comment]
Hard (BAB)
```
http://animated-pictures.net/
http://animesaprevodom.com/load/yugioh_5d_39_s/64
https://animor.tv/chuukan-kanriroku-tonegawa-009/
http://benoistbrasil.com/2018/01/27/waco-legenda-do-episodio-1x01-visions-and-omens/
https://bestnewcarsreview.com/2018-renault-alaskan-review-design-engine-release-date-photos/
```

**Comment by jspenguin2017** [comment]
Hard (BAB)
```
http://leyendas-de-occidente.blogspot.com
http://montserrat.qtellads.com/0/posts/11-China-Manufacturers-/422--Shoes-and-Accessories/
https://muktosoftware.blogspot.com/search/label/Converter
# Takes a while + a few refreshes
https://myh1z1.de/tagged/7-membersuche/?objectType=com.woltlab.wbb.thread&s=8bca277d3ad9a7b41b612385b44222f9ac03958d
# NSFW, takes a while
https://kfake.pw/2018/11/04/
```

**Comment by jspenguin2017** [comment]
Hard (BAB)
```
http://sustainable.onbeon.com/2010/11/opportunities-challenges-for.html
http://takebtc.faucethero.com/?r=1N4NRtNg8opzrG9ByJg3rCUcEGdgF7bbrr
http://tedidev.com/tag/application/page/2/
http://terror-en-el-cine.blogspot.com/2013/11/
https://tmbw.ru/nataliya-gulkina-bliny-eto-klassika
```

**Comment by jspenguin2017** [comment]
Hard (BAB)
```
http://www.xoox.co.il/clip/show_media.php?id=245
https://www.yoututosjeff.es/2018/07/crear-blog-gratis-blogger.html
# Takes a while
http://topdisegnidacolorare.biz/stampa/immagine/54-sagome-di-fiori-da-colorare-e-ritagliare-per-bambini-entro-fiori-da-colorare-per-bambini/
# Takes a while
http://wyklady.org/news/613_brakuje-ci-notatek-znamy-strone-ktora-ci-pomoze.html
```

Only 39 BAB from this scan...

**Comment by okiehsch** [comment]
`https://www.yoututosjeff.es/2018/07/crear-blog-gratis-blogger.html` is fixed in the regional list.

**Comment by jspenguin2017** [comment]
OK, added to whitelist.

**Comment by jspenguin2017** [comment]
Annoyance
```
https://80beyond.spacestation-online.com/
```

Hard
```
https://www.foguinhogames.net/
http://www.italianhotspot.com/
# Kind of hard? 
https://m5g.it
# Comes up every time
http://www.na-sportowo.com/archiwa/7216
```

**Comment by jspenguin2017** [comment]
Here's a weird one: `https://allsituspokerqq.blogspot.com/`

The page has a bug so it infinitely redirect, `allsituspokerqq.blogspot.com##+js(abort-on-property-read.js, blog)` fixes it. 

With the rule above, the page has BAB + popup.

**Comment by jspenguin2017** [comment]
That's everything except anOptions and FAB, which are always super painful to deal with. I'll leave them for another day.

**Comment by okiehsch** [comment]
Btw, thanks for all the work.

**Comment by jspenguin2017** [comment]
Thanks for your work too.

**Comment by okiehsch** [comment]
I take your point and I would not add the filter if the broken site was the only issue, but having added a filter for the anti-adblock and popup issue it seems fair to make the site usable for everyone.

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
https://www.artemia.org/
# Comes up every time
https://avenir.ro/tag/captcha/
```

Annoyance (anOptions)
```
https://www.bukmacherzy.biz/40-pln-za-zaproszenie-znajomego-w-expekt/
# Need to refresh once
http://www.beingmanan.com/wp/2011/07/unified-services-microsoft-vs-apple/
# Need to refresh once
https://www.boulderguru.de/calendar/tag_ids~247/
```

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
https://www.basentech.it/
http://fizyoo.com/tag/kas-yirtilmasina-ne-iyi-gelir/
```

Annoyance (anOptions)
```
https://eclips.ml/uncategorized/nurses-under-investigation-after-obscene-photos-with-newborns-go-viral/
https://www.historyofroyalwomen.com/
http://hollywoodredux.com/tag/luke-skywalker/
```

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
https://www.kampanyamerkez.com/tag/imzali-formaniz-kazanmakmaktan
# Need to refresh once
https://www.ilfilo.net/evento/borgo-lettura-animata-e-laboratorio-creativo/
# Redirect only happens once, but it blocks back button
https://www.ligadom.com/toros-buscan-extender-racha-leones-y-licey-quieren-sacudirse/
```

Annoyance (anOptions)
```
https://www.koragame.com/3d-air-hockey/
```

**Comment by jspenguin2017** [comment]
(ops, last post only has 4 links, I'll post 6 links here)

Hard (anOptions)
```
https://www.mkse.com/
# Comes up every time
http://www.mallorca-trend.com/tag/aktivitaten/
```

Annoyance (anOptions)
```
https://www.liguriaoggi.it/2018/10/15/principe-harry-e-megan-markle-avranno-un-royal-baby-a-primavera/
http://miph.ru/tag/video/page/2/
https://nationwideplacements.co.uk/?s=
# Need to refresh once
http://moonlightblade4fig.net/membres/isisimugu/
```

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
# Comes up every time
http://www.polizei-orchester.de/archives/category/konzerte/
# Comes up every time
http://www.portaldeabogados.net/provincia-de-buenos-aires-paro-judicial/
```

Annoyance (anOptions)
```
http://pensjonistferie.no/2017/08/24/lavkalori-brasiliansk-kokoskake/
https://www.plusdebonsplans.com/menu-restaurant-professionnels-influencent-choix-clients.html
https://revengeofthefans.com/2018/09/20/arrow-watch-a-new-trailer-for-season-7/
```

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
https://www.skinsworldtruckdriving.com.br/
# Need to refresh once
https://www.sultanarecipe.com/
```

Annoyance (anOptions)
```
https://sansmaitre.be/vivre-harmonie-cheval-dentition/dentition_cheval/
http://www.smallbizpages.ca/free-business-directory/wpbdp_tag/car-interior-upholstery/?wpbdp_sort=field-user_login
https://uvelinux.com/install-nmap-linux-ubuntu-fedora-redhat-suse/
```

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
https://www.wadalhr.com/
```

Annoyance (anOptions)
```
http://vgtribune.com/tag/marvels-spider-man/
https://wartune.dolygames.com/very-bad-news-for-gamebox-gamefuse-and-miniclip-players/
# May need to refresh once
https://thedailybounce.net/world-of-tanks/world-of-tanks-new-mechanics-for-swedish-vehicles/
# May need to refresh once
https://www.thezine.it/2017/11/17/facebook-creator-video/
```

**Comment by jspenguin2017** [comment]
Hard (anOptions)
```
# Need to refresh once
http://www.zone-musculation.com/categorymusculationpresse-a-cuisses/
```

Annoyance (anOptions)
```
https://wdbase.de/wdb/softwareprogrammierung-mit-csharp/
https://www.weedtrader.com/
http://www.zonadetrading.net/
```

Empty popup overlay
```
http://www.whoiscute.com/
```

<details>

![image](https://user-images.githubusercontent.com/7283682/50047733-a4a71680-0078-11e9-9172-2c2fac2a670b.png)

</details>

**Comment by okiehsch** [comment]
> Empty popup overlay
`http://www.whoiscute.com`

I can't reproduce, annoyance or hard?

**Comment by jspenguin2017** [comment]
@okiehsch More like ads. It's kind of a pain to remove, I'll put it into backlog.

**Comment by jspenguin2017** [comment]
Hard (FAB)
```
http://www.bbyhaber.com/bby/2018/07/11/milli-kutuphane-bakanligin-ana-hizmet-birimleri-arasindan-cikarildi
https://www.bracontece.com.br/imagem-de-gato-com-mao-humana-causa/
http://car-updates.com/2013-cadillac-cts-luxury-specs/2013-cadillac-cts-reviews-and-rating-482/
https://www.cscscholarships.net/teaching-english-in-china-esl-english-teaching-jobs-in-china-scholarship-fellow.html
https://dicariguru.com/download-rpp-bahasa-inggris-sma-smk-kelas-11-kurikulum-2013-revisi-2017/
```

**Comment by jspenguin2017** [comment]
Hard (FAB)
```
https://www.duniapelajar.com/2009/10/30/tinjauan-tentang-bank-dan-koperasi/
https://fr.thetv.info/
https://www.igli4.com/2016/09/net-framework.html
https://indofirmware.site/tag/cara-flash-vivo-y22il/
# Takes 1 refresh
https://formasyonhaber.net/-/universite-sayisi/
```

**Comment by jspenguin2017** [comment]
Hard (FAB)
```
https://iq.thetv.info/
https://klyker.com/megatron-tank-made-in-china-7-photos/
http://lgames.info/time-management/strategy-games/lost-artifacts-2-golden-island-collectors-edition.html
https://www.line-stickers.com/101-originalxjazznangostrawberry-sister/
https://mademoiselleosaki.com/dessin-a-colorier-de-vache/dessin-a-colorier-de-vache-impressionnant-coloriage-taureau-corrida-dessin-a-imprimer-gratuit-vache-beau-image/
```

**Comment by okiehsch** [comment]
`http://www.abiroyen.com` is fixed in the regional list.

**Comment by jspenguin2017** [comment]
OK, whitelisted.

**Comment by jspenguin2017** [comment]
Hard (FAB)
```
http://www.maqonsae.com/
https://www.nullwebscripts.com/blogger-templates/
http://www.pluginsxbmc.com/2013/12/presencia-en-las-redes-sociales.html
http://prawnikdomowy.info/dholki-invitation-card-wordings/
```

Annoyance (FAB)
```
http://mzk.starachowice.eu/en/
```

**Comment by jspenguin2017** [comment]
Hard (FAB)
```
https://www.reiseratgeber24.de/europa/das-ayurveda-resort-svata-katerina/27718/
https://www.synoniemboek.com/
http://treeminer.com.br/
http://ulasanseru.com/
# Alert on every refresh
http://www.trinhthamtruyen.com/la-bai-thu-xii/la-bai-thu-xii-chuong-34.html
```

**Comment by jspenguin2017** [comment]
For this one, I was able to reproduce FAB once, but now I'm getting connection error. 
```
https://driversyfirmware.com/acer-iconia-b1-720-firmware-rom-os_acer_rv0brc05-ww-gen1_a50l_a-zip/
```

**Comment by okiehsch** [comment]
`duniapelajar.com` fixed in the regional list.

**Comment by okiehsch** [comment]
`https://fr.thetv.info`
`https://iq.thetv.info`

If you have more of those, `thetv.info` will fix all of them ;)

**Comment by jspenguin2017** [comment]
That's everything from the first ~4 TB of data.

**Comment by jspenguin2017** [comment]
OK, whitelisted. 

Right now I don't.

**Comment by okiehsch** [comment]
`mademoiselleosaki.com` will be fixed separately, because of multiple anti-adblock scripts.

**Comment by okiehsch** [comment]
`http://treeminer.com.br` uses `BlockAdblock`
`http://ulasanseru.com/` fixed in the regional list.

**Comment by jspenguin2017** [comment]
`pluginsxbmc.com` `ulasanseru.com` OK, both added to whitelist. 
`driversyfirmware.com` OK, just forget about it.

**Comment by jspenguin2017** [comment]
After many failed attempts, I finally got a cluster up and running! Be ready for more results soon. 

<details>

![image](https://user-images.githubusercontent.com/7283682/50176969-0a381480-02be-11e9-8493-e2a3e171e857.png)

</details>

**Comment by jspenguin2017** [comment]
It's going a bit slower than I expected, 6450 done after about 30 hours, at this pace, it's going to run until next week. Hopefully the master server will have enough memory to sustain the leak... 

Honestly this is costing a lot, I'm still hesitating about whether to open source the scanning engine, because if I open source it, it'll be way easier for people to dodge the scan, but if I don't open source it, this will probably be an one-off thing and I won't be able to do the scanning again once my credits run out. 

I'll have another batch of scan result tomorrow afternoon, I expect it to be around 50 true positives. 

<details>

![image](https://user-images.githubusercontent.com/7283682/50260395-0e9b2500-03c5-11e9-8029-a54e0498912c.png)
![image](https://user-images.githubusercontent.com/7283682/50260576-cf210880-03c5-11e9-81c9-f88e00e84cf2.png)

</details>

**Comment by jspenguin2017** [comment]
Hard (BAB): 
```
http://2020tech.blogspot.com/
https://411newyork.org/forum/
http://www.4ufreeclassifiedads.com/-120-Ecuador-/
https://www.a100radio.com/andrew/haze-jasen-join-edge-podcast-network
https://abcgreeknews.blogspot.com/2018/05/blog-post_58.html
```

**Comment by jspenguin2017** [comment]
Hard (BAB): 
```
https://accessoirespiecesauto.net/voiture-pieces-CARBODIES-FX-FAIRWAY-2-7-TD-12653-accessoires
http://www.accessoriesfreeads.com/-93-Cook-Islands-/
http://adiccionfutbolera.com/ayudanos/
http://www.adrianocoins.com/faucet/doge206/?p=support
https://aikidochark.blogspot.com/2012/06/prueba-flash.html
https://www.ajitkumar.co.in/2018/01/automation-in-payment-of-pensionary.html
https://www.albkinema.co/
http://alienfaucet.win/
https://all4she.blogspot.com/2013/08/likoria-girls-women-disease-in-urdu-l.html
https://alldaynews.gr/paola-o-kaytos-choros-stin-pista-me-ton-niko-voyrlioti-video/
http://www.alloweb.org/french-tech/bunkr/
http://altcoingate.win/bitcoin/
https://ambbox.com/gomymobibsbs-site-theme-guide-landing-page/
http://www.amlpress.com/opinions/%D8%B9%D9%84%D9%89-%D9%82%D8%AF%D8%B1-%D8%A7%D8%A8%D8%AA%D8%B3%D8%A7%D9%85%D8%A9/
https://www.amtonline.com.br/2002/02/amt-online-news-vw-tambm-faz-recall-de.html
http://anahorny.blogspot.com/
http://androidcodelist.com/
https://www.androidfreeapk.in/tag/%E2%80%8Econtra-game-download/
http://www.androidmodapk.us/
http://www.anime4us.org/guida-le-pubblicita/
https://animeworldgreeksubs.com/category/releases/
http://www.appstorecentral.net/us/appstore/topfreeapplications
https://apptoko.com/android/last-empire-war-z/1-0-214/detail?id=com.longtech.lastwars.gp
https://www.aquitemvagas.com.br/
http://arabseries.in/En/forumdisplay.php?s=def8a70c8804c5169432a13d059b06de&f=640
http://argentina.qtelltrader.com/0/posts/41-Wholesale-Lots-/
https://www.arobasenet.com/2015/08/90-pourcent-temps-mobile-via-applis-2189.html
https://www.assistirseriesflix.net/series/assistir-serie-rebelde-Dublado-e-legendado-online/
http://aussie.totaltabs.com/
http://austria.qtelltrader.com/9--Bundesland-Tirol-/
https://www.automobiletamilan.com/news/motorshow/auto-expo/mahindra-rexton-suv-debut-at-auto-expo-2018/
https://autoonderdelen.name/auto-Gloeilamp-Achteruitrijlicht---ALFA-ROMEO-159-saloon-11842-31356-onderdelen-kar
https://autos.dailynews.us.com/2018/07/2018-ducati-scrambler-1100-sport.html
https://www.b3infoarena.in/
https://www.baderror.org/p/free-fire-mega-mod-apk-obb-data.html
http://bella-thorne.org/gallery/index.php?cat=12
http://beperfectwedding.com/wedding-dress-collection-2018-by-eddy-k/eddy-k-wedding-dress-collection-dreams-adela_full/
https://best-free-seo-tools.fatsage.com/online-url-encoder-decoder-tool
http://betterhomegardens.net/garden-design-ideas-on-a-budget
http://www.big-jobs.in/2014/09/jrengineers-electricians-welders-diesel.html
http://bitcoin.alekscoin.com/?r=1Mu3Bx82saoP3uuEBSZuyPQiWRsT4zKZxz
https://bitcoin2019.com/?p=upgrade
http://bitcoincash.alekscoin.com/?r=12x12x4fTRVA4BJpvVgcX393MYvemmpn2fmEw
http://bitcoinmoney.info/doges/?r=DCfv3qLRT2RErcKUcSrdC1XeqeuJEfHVNB
http://bks-11.com/watch-movie-the-hunger-games-19.html
http://blue-eyed-bunny.com/app_738736123_.html
https://www.borstenforum.com/topic/25191-engeltje73-7-november-2018-dr-vermeulen-st-jan-genk/?page=2
http://brd.edu.pl/testy/testy.html
https://btcearning.ambbox.com/bitcoin-earning/
https://business.dailynewsview.com/2018/10/7-signs-that-you-should-hire-pro-for.html
http://cabellopictures.com/thumbnails.php?album=638
https://caitrionabalfe.com.br/agenda/
http://camila-cabello.co/photos/displayimage.php?album=lasthits&cat=0&pid=3492
http://www.canalflix.xyz/
http://carbrandswiki.com/bugatti-veyron-best-french-car/
http://www.casinoromanesc.com/nou-portal-de-casino-online-pentru-produsele-apple-265.html
http://ccdz.tk/
https://www.cepjapones.com/
http://www.charmed-online.nl/
http://chhattisgarh.result91.com/prsu/bcom-part-3rd-exam-result-2017/70596
https://chikorita157.com/tag/pa-works/
http://chris-wood.org/photos/displayimage.php?album=282&pid=72533
http://christian.totaltabs.com/profile/?id=336746
https://www.christmasradio.live/
https://chyler-leigh.net/photos/thumbnails.php?album=toprated&cat=-377
http://cinemarcial.blogspot.com/2018/05/astro-la-nueva-pelicula-de-gary-daniels.html
https://citybicoins.website/
https://www.civilengineeringnews.tk/2014/10/stopping-signt-distance-ssd-software.html
http://claimbitz.ml/
http://claimcoin24.com/btc/?r=1Caq8aovZCczFF3ERm3mDs5ikzXLvvFUeF
http://www.classifiedsmalaysia.com/services/goods-suppliers-retailers/supermarkets.html
https://www.coachyoga.org/
http://coap.ideal.egranth.ac.in/
https://www.coasteractus.fr/2014/03/islands-of-aventures-projet-340.html
https://www.colorspire.com/color-ff6699/
http://comics-free.com/jirni-1-webp2018/
http://www.cosplay.familyrenders.com/2018/06/render-artoria.html
http://www.costapolaca.co.uk/
http://country.totaltabs.com/tablature/Kenny_Chesney/Anything_But_Mine_(solo2)_Solo_23590/
http://www.coworking-france.com/
https://cracks9.net/t/eset-nod-32-username-and-password-facebook/
https://cracxsoftwares.com/category/office/
http://cryptocurrency-world.top/doges/?r=DCfv3qLRT2RErcKUcSrdC1XeqeuJEfHVNB
http://cuciniamo.altervista.org/tag/farfalle/
http://www.culture-commune.fr/Compagnies-artistiques/compagnie-Compagnie-Sambre-Fosses-140.php
https://www.cybercommandnews.com/strange-news.html
http://czech-republic.marcyads.com/
https://www.dailyactor.com/tag/actress-emma-stone/
http://dailymalenude.com/members/2018/05/14/amateur-male-nudes-20180514-13/
http://dakotafanningbr.com/category/artigo/
http://danielmays.copperboom.us/
https://www.darklimit.ro/profile/5595-cecilv214/?status=1130&type=status
http://www.dateaprotestant.info/-346-Rwanda-/
https://dayzmap.ru/threads/iventy-ostrova-malden.916/
http://www.deadrifts.com/home/phew
http://deadsats.com/pot/?r=PUPJZ9mF74nWTVqnkRX9ASdbi1kaxff6zz
https://definicionyque.es/sindrome-del-cabeceo/
http://denmark.marcyads.com/0/posts/174-Home-Appliance-/3881--Entire-furniture-for-Sale/2694-Bambus-Messerfurnier.html
http://desimania.me/simar-and-sanjana-to-disguise-their-appearances-for-a-motive-in-sasural-simar-ka/
https://www.desiretrees.in/
https://dewhatsapp.blogspot.com/
https://www.dewijdewereld.net/reizen/wintersport-duitsland.htm
https://www.diarioherculino.com/la-vida-despues-del-vermello/
http://www.dicasdeartesanatos.com.br/2011/09/cupcake-de-fuxico.html
http://www.dinosoria.com/perrault_05.htm
https://www.directproprietar.ro/stiri
https://diytube.tk/tag/projects/
https://djdaanoliver.blogspot.com/2018/08/daan-oliver-mainstage-182.html
https://www.djeasypromos.com/
http://dl.r3sub.com/
https://www.dmznetworks.tech/2018/08/free-steam-accounts-with-games-2018.html
http://doge-coin.top/
http://doge.alekscoin.com/?r=DEuJrDQ2zLbMLX9riN5bMyLafbER3NM2rp
http://dogmoney.top/
http://dove-cameron.net/?cat=10
https://www.down-free-books.com/2016/03/download-fundamentals-of-physics.html
http://down-z.blogspot.com/2008/10/informao-warhammer-40.html
http://downloadwallpaper.org/noob-saibot-wallpaper-2/
http://www.dozarte.com/wordpress/2007/12/31/i-migliori-plugin-per-wordpress-secondo-me/?replytocom=43372
https://dreamsky10.com/wallpapers-fonts-desktop-backgrounds-fonts-and-modern-calligraphy.html/pineapple-2
http://www.drewbarrymorebr.com/gallery/
https://dubilinks.com/index.php?page=item&id=703
https://dutchycorp.space/ppc/?r=PBKAFJFVUZ86K344ZtL4mrEtC2DTLALaw8
https://earnbitcoins.fun/?r=1Kztr3ZbNnWmCmF7cJpWCSF9tWHDF869BD&rc=BTC
https://www.ebooksgospel.org/2012/11/septuaginta-guia-historico-e-literario.html
https://www.echedoros-a.gr/2018/01/blog-post_826.html
https://www.ecole-france.fr/
https://www.ecovisiones.cl/tradiciones/artic/tarot4.htm
http://ecuador.qtellfreeads.com/
http://el-cine-que-viene.blogspot.com/2018/06/tau-trailer-2018.html
https://emergencydentalcaretips.com/what-not-to-do-after-wisdom-teeth-removal/
http://emma-stone.org/photos/displayimage.php?album=lastup&cat=0&pid=10435
http://empleosmardelplata.blogspot.com/2018/08/ofertas-de-empleo-15-de-agosto.html
http://www.emplois-web.fr/emploi-startup/applications/
http://en.traxmag.com/
http://www.enalaktiko.eu/
https://www.eneskayalar.com/php-ile-instagram-kullanici-verilerini-cekmek/
https://www.enplenitud.com/dinero/cash.asp
http://epanalipseis.blogspot.com/2017/05/blog-post_73.html
https://epeliukai.lt/index.php/component/jdownloads/download/8-2-klase-lietuviu/79-pasakojimo-kurimas?Itemid=215
https://es.midiavip.com/
https://espiritudeportivo.es/noticias-huesca-futbol/7232-joge-miramon-francisco-quiere-que-seamos-valientes
http://estonia.qtellads.com/
http://euroone.dk/game/kill-the-graveyard-zombies
https://ezweightlosstips.com/everyday-tips-for-natural-and-fast-weight-loss.html
https://www.f1fanklub.pl/2016/08/f1-2016-zwiastun-premierowy-pl-gra-juz.html
http://facebumen.com/materi-tumbuh-kembang-anak/
http://fapsuey.com/asian-sex-filipina-lesbians-fuck-white-cock/
https://www.farmville2gifts.net/index.php?page=2
http://fashionbookmark.ml/user.php?login=venkintc
http://faucet.bitcoin.com.tn/
http://faucetbitcoin.tk/
http://faucetearn.com/?r=landoffaucets
http://faucetlist.win/
http://faucetprime.com/?r=landoffaucets
http://faucetranger.com/?r=landoffaucets
https://fce365.info/wp-content/hostinger-page-cache/tag/ios-11-2-1-jailbreak-kernel/page/2/_index.html
https://www.feedusgames.com/Feedus-Feed-Us-Pirates.html
https://www.fendy.web.id/?b578d5582NlM-CUSjtZd858d/ce-wd68df0b92Z-jzPQCvQXCt646c4c23.goru
https://www.ferdyhendrawan.com/
https://www.fifa-patch.com/tag/fifa-13-bundesliga-facepack-by-sergiokun16/
http://finland.marcyads.com/0/posts/152/3345--European-education-programmes/3798-STUDY-IN-ROMANIA.html
https://flalove.net/
https://www.flory4all.com/
http://www.football-streaming.co/
https://www.footbasket.com/2018/06/juju-smith-schuster-hyperx-endorsement.html
http://www.formation-culture.fr/formation-art-culture/formation-web-multimedia-graphisme-dj-network-ecole-dj-a-montpellier-formation-pour-devenir-dj-cours-mix-cours-mao/
https://forum.die-spedition.eu/partner/?s=8c70d938e434862441ba7dc594f0e21c91661f06
http://forum.italkmoney.com/statuses/user/7443-johnny/?status_id=3731
https://forum.webdeejay.it/contact/
http://forums.subglobal.net/showthread.php?tid=1743&pid=3293
https://fr.clevious.com/
https://fr.puglia-italy.co.uk/puglia-italy_hotel_directory.htm
https://francais-mahel.blogspot.com/2017/06/PesquetFLEecoutecomprehension.html
http://france-midi.blogspot.com/2017/08/den-har-gangen-klarade-vi-oss.html
http://france.marcyads.com/?view=selectcity&targetview=post&catid=136&cityid=0&lang=en
https://fraudstercrew.in/User-firas
http://fraunerd.de/pokemon-go-deshalb-sind-manche-pokemon-blau-hinterlegt/
https://freakgamers.org/forums/community.4/
http://www.freecashpotlottery.co.uk/
http://www.freepostclassifiedads.com/-430-Togo-/
https://freesmscode.com/get-virtual-phone-number-us-p-16234694485
http://fuerzasarmadas.eu/viewtopic.php?f=143&t=15664
http://www.funchocolatefacts.net/making-chocolate-covered-strawberries.php
http://www.funiforum.org/funiforum/showthread.php?t=13022&amp
http://www.funivie.org/web/tag/sicurezza-sul-lavoro/
http://www.futbolandrescl.com/
https://www.gadgetstamilan.com/news/mobiles/lephone-dazen-6a-launched-price/
http://gadvasu.bestbookbuddies.com/
http://www.gamesetorrents.com/2015/03/army-of-two-xbox360-download-torrent.html
https://www.gametork.com/
https://www.gdr-online.com/poesiedelcuore/poesia.asp?poesia=12392
http://geamap.com/taxonomy/term/37
http://geewizzapps.com/desktop/quiz-library-index.php
https://www.geologgers.com/
http://geracaorubronegra.com.br/videos/assistir-flamengo-ao-vivo-em-hd
http://www.gerapaskola.com/
https://getfreetutorial.mrunlock.pw/category/marketing/search-engine-optimization/
https://www.giochiguerra.it/gioca/tripod-attack.php
https://glossyicon.com/tag/coca-cola-icons/
http://go.paytoshi.in/
http://goldenmangas.site/
http://goowebcams.com/russia/city
https://www.gospelbook.net/2011/08/natureza-da-cura-claudinei-duarte.html
http://governmentjobindia.in/1315-ibps-specialist-officer-recruitment-2017-apply-online/
http://www.govtjobsearch.in/it-jobs/
https://grafikarsiv.com/Kategori-Yilbasi-noel
https://www.graviscomm.dk/
http://greatersudburyonline.ca/Business/TabId/130/ArtMID/596/ArticleID/677/National-Small-Business-Week-Celebrated-in-Greater-Sudbury-Bridges-to-Better-Business-Conference.aspx
http://greece.qtellfreeads.com/0/posts/135-Search-Engine-Listing-/
https://www.greecehighdefinition.com/blog/2016/12/20/ithaki-ithaca-or-ithaka-island-home-of-odysseus
http://grenada.qtellfreeads.com/
https://www.guide2dubai.com/living/expat-life/medical-tests-visa
http://gujarat.result91.com/bhavnagar-university/bachelor-of-law-llb-monsoon-sem-3rd-2018/89889
https://www.gumbleville.com/kingdom-rush
http://www.habboloader.org/
https://habr.com/post/427047/
https://www.hardwaregame.it/forum/tag/total/?s=9355172cd4b51de4b734a479669967f2
http://harmonyustv.vydio-x.com/
http://haryana.result91.com/kuk/ba-2nd-year-exam-result-april-2017/59436
https://www.hcouchd.com/
http://hddesktopwallpaper.org/high-resolution-desktop-wallpapers/
http://healthremediesinfo.com/others/health-care-delivery-system-quizlet/
http://hentaibux.xyz/?r=1Mu3Bx82saoP3uuEBSZuyPQiWRsT4zKZxz
https://www.heraldnigeria.com/
http://himachal-pradesh.result91.com/himtu/bpharmacy-1st-sem-cbcs-re-appear-exam-result-dec-2017/76106
http://www.hiringdubai.com/2018/07/manager-lean-transformation.html
http://hiringuae.com/2018/10/07/junior-butler-for-a-prince-fujairah/
http://hopqua328.com/app_916870034_.html
https://www.howunlock.com/vgotel-venture-v12-flash-file/
http://hubcoins.xyz/doges/?r=DCfv3qLRT2RErcKUcSrdC1XeqeuJEfHVNB
http://hutomosungkar.blogspot.com/
http://iantludhiana.com/?c=2
https://www.ibda3gate.com/vb/threads/%D9%84%D8%A2%D9%88%D9%84-%D9%85%D8%B1%D8%A9-%D8%A8%D8%B1%D9%86%D8%A7%D9%85%D8%AC-%D8%AA%D8%AD%D9%88%D9%8A%D9%84-%D8%B5%D9%8A%D8%BA-%D8%A7%D9%84%D9%81%D9%8A%D8%AF%D9%8A%D9%88-%D8%A7%D9%84%D8%B4%D9%87%D9%8A%D8%B1-
any-video-converter-ultimate-5-7-0-%D8%A8%D8%B7%D8%B1%D8%AD-%D9%85%D9%85%D9%8A%D8%B2-%D9%84%D9%84%D8%A8%D9%88%D8%A7%D8%A8%D8%A9.1314/
https://ibrepository.com/bl.php
https://iglo.co.in/
https://igpure.com/
https://imgrpost.com/image/a4DY3
http://www.index-of-stream.com/frame/cqtv-malaysia-s1.html
http://india.qtelltrader.com/12--Lakshadweep-/
http://indianretailsector.com/news/redmi-6a-review-specifications-analysis/
https://www.indokreasi.com/tag/enkripsi
https://indonesiapagi.com/peluang-usaha/
https://www.informagiovani-italia.com/affittare_un_appartamento_a_lisbona.htm
http://www.informatiiauto.ro/pregatirea-masinii-pentru-iarna-uid70.html
http://www.informatiquecem.com/form/index.php?forums/%D8%B3%D9%8A%D8%B1-%D8%A7%D9%84%D8%A3%D9%86%D8%A8%D9%8A%D8%A7%D8%A1-%D9%88%D8%A3%D8%B9%D9%84%D8%A7%D9%85-%D8%A7%D9%84%D8%A3%D9%85%D8%A9.289/
http://inspirates.io.ci/telechargements/details.php?file=Fantastic+Four+3D
https://www.instagram-computer.com/user_friend.php
https://www.iptvbin.com/2016/06/iptv-tom-2016-eng.html
https://iptvglobal.info/tag/atualizar-librtmp-do-kodi/
http://www.isukakipanas.com/2017/04/awaklah-adik-angkat-paling-sedap-pernah_11.html
http://www.itechnews365.com/search/label/future
https://itsjustanumber.com/profile/wooderz
http://www.itunesvideo.net/us/movies/topmovies/4432-regionalindian
https://www.jakkagiie.xyz/go.php?http://ouo.io/6c3w8f
http://januaryproxy.ml/
https://jennadewan.org/category/gallery/photoshoot/
http://jennifer-lawrence.org/index.php/2016/12/02/passengers-madrid-photocall/
http://jensenacklesbr.com/galeria/displayimage.php?pid=8222
http://www.jeux-alternatifs.com/XPtennis-jeu494_generale_1_1.html
http://jnkvv.ideal.egranth.ac.in/
http://jualtabletmurah.com/category/tips-trik/
https://www.k8-schools.com/
https://www.kafka-prag.de/startseite.html
http://kate-mckinnon.com/page/3/
https://katverse.com/06162eb49f208dd4729873d87a91aee6_large/
http://kerala.result91.com/calicut-university/1st-sem-msc-environmental-science-ccss-examination-11-2017/89358
http://kimes.info/telecharger-jeux-torrent.html
http://koha.jgu.edu.in/cgi-bin/koha/opac-detail.pl?biblionumber=191368
https://www.kolompc.us/
http://komediservisi.com/
http://www.kozmikturk.com/gundem/temizlikci-kiliginda-pkk-li-10946
http://kulinarnyduet.net/przepisy-na-pasty/
https://www.laboratoriummuzycznychfuzji.com/2012/03/eluveitie-helvetios.html
https://lagazzettadigitale.it/instagram-down-27-agosto-2018/
http://laura-prepon.net/category/oitnb/
http://lauren-german.org/
https://le-coin-de-carlos.blogspot.com/2016/01/haricots-verts-au-beurre_4.html
https://www.learn-barmaga.com/2014/05/blog-post_19.html
https://leonardolatella.it/
https://lestinet.com/site/pcholkin.com
http://www.libridaleggere.org/libri/tag/serena-vitale/
https://www.librosmedicina.org/
https://lifestyle.dailynewsview.com/
http://lights-love.com/lights-at-firefly/
https://www.limontec.com/
https://lingolex.com/italian/colours_italian.htm
https://logicalgamer.com/
http://www.logiciels.pro/comparatif-logiciels-saas-entreprise/logiciels-dintelligence-competitive/
https://www.logoinstant.com/author/aline9197552/
https://www.lokerbisnisonline.id/
https://www.lospipokos.ga/2017/04/lista-de-servidores-cs-16-2017.html
https://luchaonline.com/resultados-y-repeticion-njpw-new-year-dash-2018/
http://luke-benward.net/life-of-the-party-screencaps/
https://www.lyricsfundoo.in/2016/06/bolna-lyrics-english-translation-kapoor.html
http://maacompassion.com/Business/calibration-third-party-inspection-services/
http://madkyo.cz/component/tags/tag/106-kritika
http://maibiaoyangjia.com/liberty-land.tv.html
http://maismacetes.blogspot.com/2014/05/photoshop-cs6-chaves-de-ativacao.html
http://makemoneyonline.vydio-x.com/
http://mallinformasi.com/author/wahyudi/page/10/
http://malta.qtelltrader.com/
https://management.ind.in/forum/mah-cet-342590.html
https://www.manatelugumovies.net/category/MAA-Serials/
http://www.manchester-forum.co.uk/index.php
http://www.manualesaudi.com/
http://mcanimx.co.in/
https://medieval-engineers.eu/thread/238-server-wipe-land-of-tribulations/?postID=1270&s=94cdfc9a157260cb6aee48d0b9b1d469f8e72b66
https://www.memespuntocom.com/2018/07/resultado-de-las-elecciones-mexico-2018.html
http://memikatcahaya.com/review/software-prototyping-tool
http://www.merkuriuszkulinarny.co.uk/
http://www.merkuriuszpolonijny.co.uk/
http://meteothes.gr/wxicecoverEU.php
http://mexico.qtellads.com/0/posts/75-Jewellery-and-Clocks-/2887--Watch/
https://www.michezoafrika.com/handball/denmarks-handball-team-could-tour-kenya/4122.aspx
https://www.midiaflixhd.net/temporadas/maquina-mortifera-temporadas-2/
http://www.migranci.co.uk/2018/04/urodzinowy-obowiazek-na-emigracji.html
http://miner-faucet.tk/
http://minipazar.com/planetdoc-info.html
http://www.mishangfaucets.com/autoclaim/doge/
http://mizoram.result91.com/mzu/mba-2nd-sem-exam-result-july-2016/44326
https://mmo-population.com/r/FFXI
https://www.mobilesdownload.com/2014/08/printer-canon-pixma-mg2140-ver-150.html
https://www.moe.familyrenders.com/2018/09/konjiki-no-yami-04.html
http://monocerosrecords.com/watch-tvshow-chicago-med-236.html
http://moparconnectionmagazine.com/jacob-machowskys-1940-dodge-business-coupe/40dodge-36/
http://moviesuniversity.com/category/dualaudio/?orderby=comments
https://www.mp3virus.in/teri-aakhya-ka-yo-kajal-rajasthani-dj-uppu/
https://www.mydiploma.in/
http://mydomainworth.com/contact/
https://mylivewallpapers.com/sci-fi/pixel-cyberpunk-metropolis-animated-wallpaper/
https://www.myxclip.com/
http://nametransit.com/app_558818638_.html
http://www.nandedonline.com/
http://www.netblogbox.org/2018/06/pubg-mobile-060-apk-data-obb-for-android.html
https://newautocarpreview.com/page/99/
http://nigerianews.vydio-x.com/category/Nigeria+News
https://nightfallmoon.blogspot.com/2015/08/eden-von-hell.html
https://offbeat.dailynews.us.com/2018/11/woman-whos-in-love-with-ghost-now-ready.html
https://www.offcampusjobs4u.com/lucknow/?filter_by=featured
http://www.onlinecasinomirror.com/how-to-beat-video-poker-online-1262.html
http://oonalaurence.com/biography
http://www.ophils.com/2018/03/health-benefits-of-coffee.html
http://orealhd.blogspot.com/2014/10/enable-3d-in-photoshop-cs6-photoshop.html
https://organforum.com/forums/showthread.php?55696-Allen-TC-4-Project&s=fe81fbe085a1cf1eb8640690e3bbb483&p=491881
http://organicteaparty.org/2017/12/go-sms-pro-v744-build-419-premium-apk.html
https://osxsex.com/donna-cerca-uomo/casalinga-italiana-completissima_7006
http://www.oyp.com.ar/nueva/revistas/261/index.php
https://www.p455w0rd.net/
https://pagetrade.com/gaming/games/middle-earth-shadow-of-mordor.html
https://www.pakdigestnovels.com/
https://www.parabola.cz/prehledy/id-nagravision/
http://paraguay.qtellfreeads.com/
http://partirenthailande.fr/contact.php
https://pbclenders.com/car-loans-with-bad-credit/
https://www.pensionioggi.it/notizie/lavoro/naspi-indennita-con-limiti-per-chi-si-trasferisce-all-estero-8978979
https://pentafaucet.com/?r=1Mu3Bx82saoP3uuEBSZuyPQiWRsT4zKZxz
https://photos.dailynews.us.com/2018/11/photos-of-day-october-31-2018.html
https://photos.dailynewsview.com/2018/09/photos-of-day-september-11-2018.html
https://www.photostatouages.com/dessins-de-tatouages/
https://www.phrepublic.com/2018/08/disney-junior-play-v140-mod.html
https://pieseautoonline.org/model-piese-auto-DAEWOO-LACETTI-hatchback-KLAN--5046
https://www.pijarkita.com/2014/12/baca-buku-gratis-ilmu-pengetahuan-alam.html
http://www.pinklinkz.com/category/bukkake/
http://pinoytvph.net/
https://pleximum.com/dl/?f=SteinsGT0_DUB_15.7z
http://www.pokemontowerdefense.net/pokemon-trading.htm
http://pokexgames-tutoriais.blogspot.com/2017/12/evento-natal-2017.html
https://politics.dailynews.us.com/2018/05/trumps-fluid-approach-to-national-and.html
http://pornparadise.org/x-art-stephanie-amaris-hot-fitness-sex-19-07-2018-lesbian-1080p/
http://portableshare.blogspot.com/2014/01/ultraiso-premium-edition-961-build-3016.html
http://portugal.marcyads.com/0/posts/196-Stamps-and-Coins-/4262--Coins-for-Sale/
http://www.possoftwaresystem.org/
http://post.qtellads.com/0/events/2018-10-20/
http://postfreeclassifieds.qtellads.com/-218-Kiribati-/
https://www.primerdiscurso.com/
http://www.proxyfree.co/
http://proxyrio.net/
http://publishersarchive.com/publisher.php?pub_id=11561
http://pvnrtvu.ideal.egranth.ac.in/cgi-bin/koha/opac-authorities-home.pl
http://www.qtellfreedownloadtrader.com/0/events/2018-06-16/
http://www.qtellnet.com/0/events/2018-07-30/
http://www.quantobaste.com/
https://www.ragcomercio.com/
http://www.railwayoperationsimulator.com/railway-projects/shanghai-metro
http://rajasthan.result91.com/uor?page=4
http://ramazanayi.blogspot.com/
https://rapidleecher.cf/
https://www.rc-help.com/
https://readingpk.com/butshikan/
http://redtube-cs.tk/
http://revistastejidosymanualidades.blogspot.com/2011/12/revista-bolsos-crochet-paso-paso.html
https://www.revopowaaa.com/t3072p50-erbe-config-bash-solide-6s-2200kv-mamba-de-truggy-p
https://ripplenews.tech/2018/06/15/ripple-representative-finds-banks-unlikely-to-use-blockchain-for-cross-border-payments/
http://rock.totaltabs.com/tablature/ACDC/Ride_On_Chord_65738/?h=&exp=
https://rumahfilm21.web.id/
https://ryo.altfaucet.xyz/
https://www.sachonthi.com/
http://www.sacredwiki.org/index.php/SacredWiki
https://sadiesink.net/photos/displayimage.php?pos=-638
https://www.safaxnet.com.ng/2017/08/jumia-deals-of-day-26-august-2017.html
http://www.sambafoot.co.uk/heading-games.html
https://saradahentai.com/tag/katarina/
https://sarapbeh.com/malaking-pwet-na-girl/
https://www.sarkari-naukrialert.com/2017/07/spmcil-recruitment-2017-supervisor-vacancies.html
http://www.sarkarinaukriok.in/
http://satobitties.com/
https://scanfact.blogspot.com/2018/09/khloe-kardashian-tells-kim-and-kourtney.html
https://www.scaricaimmagini.it/master/ALTRE/Section-4/Wallpapers-87/index.php
http://www.sclubmedia.net/wp/sc3-family-teaser-2/
https://screwcams.com/videos/alesya-romanian-cam-whore-nude-sex-cams-free/
https://www.seebeetee.com/why-mobile-learning-applications-are-future-of-education/
https://sem40.co.il/307072-amerikanskie-ravviny-ne-puskayut-v-sinagogi-teh-kto-ne-privit-ot-kori.html
http://serbia.marcyads.com/
http://serbia.qtellfreeads.com/
http://server.satunivers.tv/gc/index.html
http://sexxo.info/?url=aHR0cHM6Ly91cHRvYm94LmNvbS9wZ2Y2NW50MW1od28
http://sharkfaucet.org/
http://www.sharkgame.net/hacked-shark-games.html
https://shemalepower.xyz/centoxcento-strane-voglie/
https://showboxtvapp.com/privacy-policy/
http://shreecrooks.com/gallery/displayimage.php?album=lastup&cat=14&pid=965
http://www.sifthead.net/
https://signals.easyindicators.com/web/easymacdx/
https://www.significatocanzoni.it/
http://sknau.ideal.egranth.ac.in/cgi-bin/koha/opac-main.pl
http://slovakia.qtellads.com/0/posts/10-Children-and-Babies-/348--Feeding/
http://smoothtreats.com/tag/red-2010-movie-download/
http://snackfaucets.xyz/faucetlist.php
http://sooperonlinejobs.blogspot.com/2016/10/earn-money-from-kolotibablo-referral-program.html
https://space-engineers.de/?s=a641ab528ce8f1c089c823a49f682aaef8118995
https://sports.dailynews.us.com/2018/11/washington-capitals-begin-busy-stretch.html
https://sports.dailynewsview.com/2018/09/big-ten-network-suspends-braylon.html
http://spotgif.com.br/picture.php?/2185/categories
http://squirtfix.com/category/bbc/
http://sriluseo.tk/new.php
http://www.ssc-cgl2014.in/2016/
https://www.sshfunkysst.net/
http://stars8doge.win/
http://www.startups-news.fr/communique-presse/startups-services/startups-administration/startups-droit/startups-legal/
http://www.stem-cells-news.com/1/the-fda-is-killing-adult-stem-cell-therapy-thus-killing-patients/
http://stephenamellsource.fanhost.net/
https://store.kodular.io/home
https://studychacha.com/discuss/394122-ifsc-code-south-indian-bank.html
https://www.stunmore.com/toro-rosso.html
http://www.subglobal.net/
http://sunnysideshlee.com/
http://sweden.marcyads.com/
http://swiftclassifieds.com/pets_goa-r781500
http://switzerland.qtellads.com/0/posts/6-Body-Care-/102--Face-Care/
https://www.syreencartoon.com/category.php?cat=kasi-and-leuk
http://syria.qtellads.com/
http://www.tableinduction.fr/
http://www.talpo.it/
http://tamercome.blogspot.com/2014/10/dil-ki-baazi-1993-dvdrip-480p-x264-ac3.html
https://www.tech-gold.net/
https://www.technicalbaba.com/app/all-questions
https://www.tectuto.com/
http://telangana.result91.com/sbtetap-tg/ts-sbtet-diploma-c14-exam-result-oct-nov-2016/49256
http://www.televisiondominicanaenvivo.com/
https://tempmail.in/
https://www.th-world.com/2018/10/Download-game-hello-stars.html
https://www.the-prof-dz.com/p/site-map.html
http://theblueunderground.com/
https://www.thecraftbooth.com/write-for-us/
http://theengineeringmindset.com/tag/food-retail/
https://www.themebanks.com/download-advanced-floating-sliding-panel/
https://thereadingruby.blogspot.com/2018/01/how-to-fall-in-love-with-man-who-lives.html
http://thirdistudio.com.au/DIY%20Pallet%20Wood%20Computer%20Desk/PalletWoodComputerDesk_HowtoCutWetDryPalletWood.html
http://www.tiktik.net/
http://tipdad.com/
http://www.tocadopip.com/2017/01/politica-de-privacidade.html
https://tokeativity.com/connect/tags/south%20africa/
http://toodnews.blogspot.com/2016/08/03082016.html
https://top.howfn.com/2017/11/bank-check-book-facility-close-in-near.html
http://top5identitytheftprotection.com/planet-series.html
https://www.tormalayalam.com/2017
https://tpb.crushus.com/audioz.cc/2017/03/
https://travel.dailynews.us.com/2018/05/southwest-launches-four-day-sale-with.html
http://trbet206.com/planetdoc-info.html
http://trets.free.fr/actu/agendatretsois.php
https://www.trumpservativenews.club/2018/11/william-murchison-calling-george-mason.html
https://www.tudoengcivil.com.br/2014/10/triangulacao-de-gauss-exercicios.html
https://tunisie-travail.tn/sinscrire/?action=register
https://www.tutelobajas.com/descargar-afterfocus-pro-logra-el-efecto-del-iphone-7-en-android/
http://tutorialsnation.com/page/3
https://tutuapp.pro/
http://www.twitterslut.org/2018/07/super-natural-wonders-eva-berg.html
http://twohundredone.com/index.php?device=1&c=18&p=1
http://uae-careers.blogspot.com/
http://uaytcn.biz/app_1251282152_.html
https://ufoport.com/onlajn-gadaene-bezplatno/
http://united-kingdom.marcyads.com/0/events/2018-06-15/
http://united-kingdom.qtelltrader.com/?view=post&postevent=1&cityid=101&lang=en&catid=&subcatid=
http://united-states.qtelltrader.com/-19-Delaware-/
http://www.uponedroid.com/2015/05/moto-g-2014-e-moto-x-2014-comecaram-a-receber-o-android-51.html
https://www.uppic.org/
http://urbanlove.ru/articles/jenskie-interesi/jenshina-za-rulem-kak-ne-boyatsya-sest-vpervie/
https://urdu.i360.pk/%D9%85%D8%B9%D9%88%D8%B0%D8%AA%DB%8C%D9%86-%DA%A9%D8%A7-%D9%85%D8%BA%D8%A7%D9%84%D8%B7%DB%81-%D8%A7%D8%B2-%D9%82%D8%A7%D8%B1%DB%8C-%D8%AD%D9%86%DB%8C%D9%81-%DA%88%D8%A7%D8%B1/
https://us.dailynews.us.com/2018/07/formula-e-eprix-returns-to-red-hook.html
https://us.midiavip.com/
http://uttar-pradesh.result91.com/kgmu/mbbs-first-professional-regular-exam-result-2018-mlnmc-allahabad/89802
https://www.vegam.today/lifestyle/travel/5176/unpacking-that-suitcase/newsbestsite/
https://vepornhd.club/if-the-shoe-fits-monica-asis-jmac/
http://www.victorharunasblog.com.ng/p/about.html
http://www.videohits.tv/journey-girl-cant-help-it-video_4e50acea3.html
http://videominecraft.vydio-x.com/
http://videos-marciales.blogspot.com/2009/
https://vidgood.com/bride-wars-deadly-beloved-comedy-central_fb5ede35e.html
https://viesosiosrimtiestrikdymas.blogspot.com/
https://www.vindexcraft.net/
http://vintagewinepicks.blogspot.com/2018/11/bottega-accademia-red-prosecco-italy-wine-review.html?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+VintagesWinePicks+%28VINTAGES+Wine+Picks+%26+Reviews%29
http://www.vocaloid.familyrenders.com/2017/06/render-mayu_9.html
http://www.volosfans.com/2016/11/blog-post_10.html
http://www.voyageguineeconakry.com/
https://vpsservs.com/
http://w1.watch7deadlysins2.net/7dss/
http://www.walkingforum.co.uk/index.php?topic=881.msg5033
http://wallpapermad.com/hip-hop-wallpaper/
http://www.wartaperang.com/2016/03/bom-mobil-kedua-dalam-sebulan-34-tewas-turki.html
https://watchcharmedonline.com/episodes/charmed-8x2/
https://watchpinoymoviesonline.info/just-the-3-of-us-2016/
https://watchthementalistonline.com/seasons/the-mentalist-season-6/
https://www.wendigostore.com/
http://west-bengal.result91.com/burdwan-university/admission-mphil-programme-session-2018-20/86781
https://whatdownloadmeans.blogspot.com/2018/08/sword-man-monster-hunter-mega-mod-x10.html
https://winasweepstakes.com/au/prize-type/win-cash-and-gift-cards
https://www.winauthority.com/2017/11/30/man-deactivated-donald-trumps-twitter-reveals/
http://www.windows-driver.org/2016/11/25/hp-deskjet-6540-printer-driver-2/
https://world.dailynewsview.com/2018/10/malaysia-ex-pms-wife-pleads-not-guilty.html
http://ww1.heroacademia2.net/
http://wynalazki.andrej.edu.pl/index.php/wynalazki/12-c/89-czarna-skrzynka
https://www.x.familyrenders.com/2018/05/render-jessie-maye.html
https://xda-app-developers.blogspot.com/2018/07/samsung-galaxy-note-9-s-pen-could-be.html
http://xinyu668.com/app_1020541183_.html
http://www.xss-server.com/index.php?/index.php
https://yescams.com/videos/tag/amateur/page/11/
https://www.yggtorrent.fr/telecharger-le-torrent-ralph-2.html
http://yourbookmark.tk/?part=commented&category=News
http://www.yuri.familyrenders.com/2018/05/render-amemiya-hatsune-takashiro-nana.html
http://zendaya-coleman.org/gallery/displayimage.php?album=lasthits&cat=0&pid=449
https://www.ziarelive.ro/stiri/ministrul-rus-al-apararii-tensiunile-dintre-sua-si-rusia-au-fost-alimentate-artificial.html
http://www.zverinova-kucharka.cz/?option=com_search&searchword=pol%C3%A9vka
```

**Comment by jspenguin2017** [comment]
As far as I see, BAB scanning is pretty accurate, there's over 500 results, I don't want to test each of them... 

I checked about 10 of them.

**Comment by okiehsch** [comment]
I will also not test all of them .....

**Comment by jspenguin2017** [comment]
I expect over 95% accuracy, considering past results.

**Comment by okiehsch** [comment]
Well, `bab-defuser` will not break anything so that is acceptable in my opinion.

**Comment by jspenguin2017** [comment]
Hard (app_vars): 
```
http://www.7aytna.com/THq2
http://9ig.de/09bNaDd
https://abshort.co/25aFC
https://www.addslinkearn.com/InaJMx3G
https://adsnet.com.pe/6WG3
```

**Comment by jspenguin2017** [comment]
Hard (app_vars): 
```
https://btc4link.com/SOZAABDonate
https://egypt-mix.com/10KM7w05
http://www.fastmoney71.com/6
http://hotbitco.win/UbDOPi
http://inurls.co/JaNkiCz
```

**Comment by jspenguin2017** [comment]
Hard (app_vars): 
```
https://link.anhdep24.net/1nBHszcIPp
https://link.clip4all.com/3g2L4dD
http://link2short.com/A3aVLnU
https://lwt.pw/Gd5haf
https://myshrinker.com/h5NN
```

**Comment by jspenguin2017** [comment]
Hard (app_vars): 
```
http://novaurl.com/I5gvNK1z
https://owr.io/5VXnn3
http://pastecash.com/GO3Ad
https://rgl.vn/cNY8
http://s.sseluxx.com/MkjYgr
```

**Comment by jspenguin2017** [comment]
Hard (app_vars): 
```
https://shln.pw/6SNnL5
http://short.gs/YNb
http://shortens.co/20VZYZgo
https://www.wolfshort.com/i30UF5
http://tinylink.run/4QuoFFD
```

**Comment by jspenguin2017** [comment]
Hard (app_vars):
```
http://youlink.ga/Pexa
```

Enjoy your daily dose of link shorteners 😄

**Comment by okiehsch** [comment]
Great the first one of that batch is already a special case.

**Comment by jspenguin2017** [comment]
Take your time, there's no rush on this. Don't burn yourself out!

**Comment by jspenguin2017** [comment]
Hum... We need a new strategy. 

![image](https://user-images.githubusercontent.com/7283682/50360192-69936000-051c-11e9-86e7-160dde16658a.png)
![image](https://user-images.githubusercontent.com/7283682/50360398-1a99fa80-051d-11e9-9a26-f6ed1198a300.png)

This thread can be closed when those [`app_vars`](https://github.com/uBlockOrigin/uAssets/issues/4268#issuecomment-449131154) are fixed. 

Work smart, not hard. I need to think about the next step...

**Comment by okiehsch** [comment]
Removed `habr.com`.

**Comment by jspenguin2017** [comment]
I thought the chance that `bab-defuser` break stuff is slim enough that we can get away with it, but apparently not... I'll be using Puppeteer to test all BAB domains that were not tested: 
https://github.com/uBlockOrigin/uAssets/issues/4368#issuecomment-449581347
https://github.com/uBlockOrigin/uAssets/issues/4268#issuecomment-449126844
https://github.com/NanoMeow/QuickReports/issues/178#issuecomment-427983420

**Comment by reza41** [comment]
3d eart scanner
